modules/ezcontent_block/modules/ezcontent_block_alert_banner/src/Plugin/Field/FieldFormatter/AlertBannerTextDefaultFormatter.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Drupal\ezcontent_block_alert_banner\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Field\FormatterBase;
+use Drupal\Core\Field\FieldItemListInterface;
+
+/**
+ * Plugin implementation of the 'alert_banner_text_default' formatter.
+ *
+ * @FieldFormatter(
+ *   id = "alert_banner_text_default",
+ *   label = @Translation("Alert Banner"),
+ *   field_types = {
+ *     "text",
+ *     "text_long",
+ *     "text_with_summary",
+ *   }
+ * )
+ */
+class AlertBannerTextDefaultFormatter extends FormatterBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function viewElements(FieldItemListInterface $items, $langcode) {
+    $elements = [];
+    $entity = $items->getEntity();
+    foreach ($items as $delta => $item) {
+      $elements[$delta] = [
+        '#type' => 'processed_text',
+        '#text' => $item->value,
+        '#format' => $item->format,
+        '#langcode' => $item->getLangcode(),
+      ];
+    }
+    $bg = $entity->get('field_background_color')->first()->getValue();
```

Comment:
The Background color field is non mandatory - https://github.com/srijanone/ezcontent/pull/230/files#diff-75df8b443deb2f25742405740037b988R15
I believe, we should handle this here.

Reply:
As discussed, this would return the default value, if left empty by user.